css/desktop.css

```diff
@@ -33,6 +33,13 @@
     color: #344563;
   }
 
+  .links a {
```

Comment:
- [ ] Kindly add the horizontal line separation between the languages and the links just like in the figma design. 
- [ ] Add the icons for "See Live" and "See Source" links. Please also make sure that the text in these links don't stay in the same line.

Fix these issues and your project is good to go.
Cheers 👍
  
Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/179983464-005eb242-60a6-467b-9b63-c8e940d57417.png)  |  ![image](https://user-images.githubusercontent.com/60618877/179983545-f17a3979-0c64-4a1b-ab5d-c20674652464.png)